tests/integration/components/truncate-multiline-test.js

```diff
@@ -578,3 +579,25 @@ test('changing the component\'s lines changes the resulting markup', function(as
     'new number of chunks is respected'
   );
 });
+
+test('<BR>s are replaced with spaces in truncated output', function(assert) {
+  assert.expect(1);
+
+  this.set('lineToTruncate', 1);
+  this.set('text', Ember.String.htmlSafe('supercalifragilisticexpialidocious<br>supercalifragilisticexpialidocious'));
```

Comment:
Can you please add a set of 2+ adjacent `<br>` tags? Since there are currently no tests for the clamp utility, I'd like to ensure this test covers the possible scenarios of https://github.com/nickiaconis/ember-truncate/pull/21/files#diff-d959edf92a5bb906228211dd3539015dR78